yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -1114,13 +1147,41 @@ func fetchUnsupportedObjectTypes() ([]UnsupportedFeature, error) {
 			unsupportedIndexes = append(unsupportedIndexes, ObjectInfo{
 				ObjectName: fmt.Sprintf("Index Name: %s, Index Type=%s", objectName, objectType),
 			})
+
+			assessmentReport.AppendIssues(AssessmentIssue{
```

Comment:
Impact is not added for these issues in this function. TODO?

Reply:
added a comment
```
// For these oracle issues defining Impact not required, since oracle migration complexity comes from ora2pg
```